archaius2-test/src/test/java/com/netflix/archaius/test/Archaius2RuleTest.java

```diff
@@ -18,14 +21,28 @@ public void testBasicPropertyResolution() {
         assertEquals("present", config.getString("parentClassLevelProperty"));
         assertEquals("present", config.getString("classLevelProperty"));
         assertEquals("present", config.getString("testLevelProperty"));
+        assertEquals("type", config.getString("cascading"));
```

Comment:
Consider adding a unit test to verify the fix for the class annotation ordering issue I mentioned earlier.